tutorials/AMB2018-02-B/Allrun

```diff
@@ -3,13 +3,36 @@ cd ${0%/*} || exit 1    # Run from this directory
 
 # Source tutorial run functions
 . $WM_PROJECT_DIR/bin/tools/RunFunctions
-
 application=`getApplication`
 
+# Parse arguments
+withExaCA=false
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    -withExaCA)
+      withExaCA=true
```

Comment:
You could automatically uncomment the FO include line in `controlDict` when the `-withExaCA` flag is set to simplify the workflow for users.

Reply:
fixed: 823f8121c12068307795d1ba20561fb7ce93fe95